packages/core/src/Stores/client-store.js

```diff
@@ -615,6 +618,129 @@ export default class ClientStore extends BaseStore {
         return this.all_loginids.map(id => this.getAccountInfo(id)).filter(account => account);
     }
 
+    get wallet_list() {
```

Comment:
We dont need to add more load to client store. You can create a hook in the hooks package. Same goes to the bottom 2 functions